tests/backends/oracle/test_introspection.py

```diff
@@ -33,3 +33,27 @@ def test_get_sequences_manually_created_index(self):
                 # Recreate model, because adding identity is impossible.
                 editor.delete_model(Square)
                 editor.create_model(Square)
+
+    def test_get_table_description(self):
+        with connection.cursor() as cursor:
+            # First, we create a test table
+            cursor.execute(
+                """
+                CREATE TABLE TEST_TABLE (
+                    TEST_FIELD VARCHAR2(100) COLLATE BINARY_CI
+                )
+            """
+            )
+            try:
+                # Then we get the description of the table
+                description = connection.introspection.get_table_description(
+                    cursor, "TEST_TABLE"
+                )
+
+                # Finally, we make assertions about the table description
+                self.assertEqual(len(description), 1)
+                self.assertEqual(description[0][0], "TEST_FIELD")
+                self.assertEqual(description[0][1], "VARCHAR2")
+            finally:
+                # Clean up after ourselves by removing the test table
+                cursor.execute("DROP TABLE TEST_TABLE")
```

Comment:
This ticket is about collations for view and materialized view, not for a table. Introspection for tables is already covered by `inspectdb.tests` and works fine, this is not a proper regression test. Do you have an Oracle test environment?

Reply:
This is an example of a regression test for this issue:
```diff
diff --git a/tests/backends/oracle/test_introspection.py b/tests/backends/oracle/test_introspection.py
index c6a143eb2d..e75d3a6c50 100644
--- a/tests/backends/oracle/test_introspection.py
+++ b/tests/backends/oracle/test_introspection.py
@@ -3,7 +3,7 @@ import unittest
 from django.db import connection
 from django.test import TransactionTestCase
 
-from ..models import Square
+from ..models import Person, Square
 
 
 @unittest.skipUnless(connection.vendor == "oracle", "Oracle tests")
@@ -33,3 +33,20 @@ class DatabaseSequenceTests(TransactionTestCase):
                 # Recreate model, because adding identity is impossible.
                 editor.delete_model(Square)
                 editor.create_model(Square)
+
+    def test_get_table_descriptioni_view_default_collation(self):
+        person_table = connection.introspection.identifier_converter(
+            Person._meta.db_table
+        )
+        person_view = connection.introspection.identifier_converter("TEST_PERSON_VIEW")
+        with connection.cursor() as cursor:
+            cursor.execute(f"CREATE VIEW {person_view} AS SELECT * FROM {person_table}")
+            try:
+                description = connection.introspection.get_table_description(
+                    cursor, person_view
+                )
+                self.assertEqual(len(description), 3)
+                for column in description:
+                    self.assertIsNone(column.collation)
+            finally:
+                cursor.execute(f"DROP VIEW {person_view}")
```